kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -13,9 +13,32 @@ This example shows
 * working with user tasks
 * four eye principle with user tasks
 	
-	
+* Diagram	
```

Comment:
not present into https://github.com/kiegroup/kogito-quickstarts/blob/f19e115ba63b96c375caa81f092c5ea71bb411f3/kogito-usertasks-quarkus/README.md
can be removed

Reply:
fixed @radtriste